src/tests/functional/shared_test_classes/include/shared_test_classes/base/layer_test_utils.hpp

```diff
@@ -160,7 +160,7 @@ class LayerTestsCommon : public CommonTestUtils::TestsCommon {
     InferenceEngine::Layout inLayout = InferenceEngine::Layout::ANY;
     InferenceEngine::Layout outLayout = InferenceEngine::Layout::ANY;
     InferenceEngine::Precision inPrc = InferenceEngine::Precision::UNSPECIFIED;
```

Comment:
1. The same mechanism should be implemented for `inPrc` in both classes.
    Looks like and for layouts too
2. The vector size should be aligned with input/output counters.
    I thinks the best option here is not to initialize the vector here. I think we should use  `InferenceEngine::Precision::UNSPECIFIED` in `configureModel()` if the vector is empty.

Reply:
1. `inPrc` is vectorized in commit https://github.com/openvinotoolkit/openvino/pull/9975/commits/184a2417a8ad90cbd3faae13dceddd0866e7acad .
`inLayout` and `outLayout` are vectorized in commit https://github.com/openvinotoolkit/openvino/pull/9975/commits/0e8d906b307afed2b71eaf191508a1e403452d55 and https://github.com/openvinotoolkit/openvino/pull/9975/commits/dd296a308f103f141905b2abbcfb3548f8d93fdb .
2. The vector initialization part is currently moved to construct function, commit https://github.com/openvinotoolkit/openvino/pull/9975/commits/92114ec114f55a343265a2ed9d8f29c7d0f65c4e . This is because these vectors are used in `SetUp` function in single layer test, and initialize them in `configure_model`/`ConfigureNetwork` is too late.